src/coreclr/pal/src/thread/process.cpp

```diff
@@ -2399,16 +2431,36 @@ PROCCreateCrashDump(std::vector<const char*>& argv)
         {
             // Ignore any error because on some CentOS and OpenSUSE distros, it isn't
             // supported but createdump works just fine.
-            ERROR("PROCCreateCrashDump: prctl() FAILED %d (%s)\n", errno, strerror(errno));
+            ERROR("PROCCreateCrashDump: prctl() FAILED %s (%d)\n", errno, strerror(errno));
```

Comment:
Did you forget to swap `errno` and `strerror(errno)`?

Reply:
Oops, looks like I did. I'll fix it in a future PR.